lib/step.js

```diff
@@ -95,6 +95,9 @@ class Step {
 
   /** @return {string} */
   humanizeArgs() {
+    if ( !args ) {
```

Comment:
this.args

Reply:
I see, updated method now, pls check again

Reply:
hm... interested, why some CI jobs not started)

Reply:
yup, I noticed that too. Is there some log available to investigate the cause?